src/core/qgsvectorfilewriter.cpp

```diff
@@ -1079,6 +1079,13 @@ class QgsVectorFileWriterMetadataContainer
                              15 // Default value
                            ) );
 
+      layerOptions.insert( QStringLiteral( "RFC7946" ), new QgsVectorFileWriter::BoolOption(
+                             QObject::tr( "Whether to use RFC 7946 standard. "
```

Comment:
```suggestion
                             QObject::tr( "Skip CRS creation (RFC 7946). "
```

I think it won't hurt to be verbose about what this option does. It's easier for humans to anticipate the implication like this I would say.